lib/player.js

```diff
@@ -1777,6 +1777,14 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     // Since we are about to start playback, we will lock in the start time as
     // something we are now depending on.
     has.startTime = wants.startTime;
+    console.info('onLoad_() currentTime=' + has.mediaElement.currentTime +
+        ' startTime=' + has.startTime);
```

Comment:
I believe console.info will be banned by our compiler settings.  Can you please remove this and other debugging cruft?  If you want to log this permanently, use `shaka.log`, which we can control at compile-time for production and runtime for debug builds.